spec/models/classified_listing_spec.rb

```diff
@@ -71,4 +71,46 @@
       expect(credit.reload.purchase).to be_nil
     end
   end
+
+  describe "#index_to_elasticsearch" do
+    it "enqueues job to index classified_listing to elasticsearch" do
+      sidekiq_assert_enqueued_with(job: Search::ClassifiedListingEsIndexWorker, args: [classified_listing.id]) do
+        classified_listing.index_to_elasticsearch
+      end
+    end
+  end
+
+  describe "#index_to_elasticsearch_inline" do
+    it "indexed classified_listing to elasticsearch inline" do
+      allow(Search::ClassifiedListing).to receive(:index)
+      classified_listing.index_to_elasticsearch_inline
+      expect(Search::ClassifiedListing).to have_received(:index).with(classified_listing.id, hash_including(:id, :body_markdown))
+    end
+  end
+
+  describe "#after_commit" do
+    it "enqueues worker to index tag to elasticsearch" do
+      classified_listing.save
+
+      sidekiq_assert_enqueued_with(job: Search::ClassifiedListingEsIndexWorker, args: [classified_listing.id]) do
+        classified_listing.save
+      end
+    end
+  end
+
+  describe "#serialized_search_hash" do
+    it "creates a valid serialized hash to send to elasticsearch" do
+      # Sorting keys to ignore order
```

Comment:
Could also do something a little more generic like using `include`, but I kind of like how explicit this is.

Reply:
Here you can also use `match_array` Per [the docs](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers), match_array will ignore the order for you. 
```
expect([1, 2, 3]).to     match_array([3, 2, 1])
```